### PR TITLE
mgr/dashboard: Update list of supported browsers

### PR DESCRIPTION
**Pull request by tspmelo**
Fixes: https://tracker.ceph.com/issues/46923

Signed-off-by: Tiago Melo <tmelo@suse.com>

## Checklist
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tspmelo** [comment]
jenkins render docs

File: doc/mgr/dashboard.rst [hunk 1]
```diff
@@ -131,13 +131,15 @@ Supported Browsers
...
-|                    Browser                    | Versions |
-+===============================================+==========+
-| `Chrome <https://www.google.com/chrome/>`_    | 68+      |
-+-----------------------------------------------+----------+
-| `Firefox <https://www.mozilla.org/firefox/>`_ | 61+      |
-+-----------------------------------------------+----------+
++-------------------------------------------------------------------------+--------------------+
+|                                 Browser                                 |      Versions      |
++=========================================================================+====================+
+| `Chrome <https://www.google.com/chrome/>`_ and chromium based browsers  | latest 2 versions  |
```
Comment (s0nea) [inline doc/mgr/dashboard.rst:137]:
```suggestion
| `Chrome <https://www.google.com/chrome/>`_ and `Chromium <https://www.chromium.org/>`_ based browsers  | latest 2 versions  |
```

Comment (tspmelo) [reply doc/mgr/dashboard.rst:137]: done

**Comment by tspmelo** [comment]
jenkins render docs

File: doc/mgr/dashboard.rst [hunk 2]
```diff
@@ -131,13 +131,16 @@ Supported Browsers
...
-|                    Browser                    | Versions |
-+===============================================+==========+
-| `Chrome <https://www.google.com/chrome/>`_    | 68+      |
-+-----------------------------------------------+----------+
-| `Firefox <https://www.mozilla.org/firefox/>`_ | 61+      |
-+-----------------------------------------------+----------+
++---------------------------------------------------------------+----------------------------+
+|                            Browser                            |          Versions          |
++===============================================================+============================+
+| `Chrome <https://www.google.com/chrome/>`_ and                | latest 2 versions          |
```
Comment (LenzGr) [inline doc/mgr/dashboard.rst:137]: I think we should be more specific here:
```suggestion
| `Chrome <https://www.google.com/chrome/>`_ and                | latest 2 major versions    |
```

File: doc/mgr/dashboard.rst [hunk 3]
```diff
@@ -131,13 +131,16 @@ Supported Browsers
...
+| `Chromium <https://www.chromium.org/>`_ based browsers        |                            |
++---------------------------------------------------------------+----------------------------+
+| `Firefox <https://www.mozilla.org/firefox/>`_                 | latest 2 versions          |
```
Comment (LenzGr) [inline doc/mgr/dashboard.rst:140]: Dito here:
```suggestion
| `Firefox <https://www.mozilla.org/firefox/>`_                 | latest 2 major versions    |
```

**Pull request merged**